framework/src/media/FocusRequest.cpp

```diff
@@ -16,21 +16,33 @@
  *
  ******************************************************************/
 
-#include <sstream>
 #include <media/FocusRequest.h>
 
 namespace media {
 std::string FocusRequest::getId()
 {
-	return mId;
+	return std::to_string(mStreamInfo->id);
+}
+
+bool FocusRequest::compare(FocusRequest &focusRequest)
+{
+	return this->mStreamInfo->id == focusRequest.mStreamInfo->id;
+}
+
+bool FocusRequest::compare(std::shared_ptr<FocusRequest> focusRequest)
```

Comment:
This is builder class, please think about below.
1. Rename FocusRequest.cpp to FocusRequestBuilder.cpp
2. It is very hard to treat 'compare' as a feature of request builder. I think this can be moved to focusManager, also this may used in there.

Reply:
`FocusRequest` has an inner class called `Builder`.
So it is been used like `FocusRequest::Builder builder`.

Since `FocusRequest::Builder` is a part of `FocusRequest`, the cpp file name should be FocusRequest.cpp